tests/integration/test_api_gateway.py

```diff
@@ -123,6 +127,16 @@ def forward_request(self, **kwargs):
         proxy.stop()
 
     def test_api_gateway_lambda_proxy_integration(self):
+        self._test_api_gateway_lambda_proxy_integration(
+            self.TEST_LAMBDA_PROXY_BACKEND,
+            self.API_PATH_LAMBDA_PROXY_BACKEND)
+
+    def test_api_gateway_lambda_proxy_integration_with_path_param(self):
+        self._test_api_gateway_lambda_proxy_integration(
+            self.TEST_LAMBDA_PROXY_BACKEND_WITH_PATH_PARAM,
+            self.API_PATH_LAMBDA_PROXY_BACKEND_WITH_PATH_PARAM)
+
+    def _test_api_gateway_lambda_proxy_integration(self, _fnName, _path):
```

Comment:
nitpick: we usually use `under_score` variables instead of `camelCase`. Also, parameters usually don't start with an underscore. Not critical, but something to bear in mind if we want to adhere to the same style guidelines that are already used in the project.